CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ The `firebase_js_interop` package provides direct access to the Firebase JS SDKs
 
 The process for adding new interop code is as follows:
 
-1. Find the relevant code in the JS SDK
+1. Find the relevant code in the [JS SDK](https://github.com/firebase/firebase-functions/tree/master/src)
```

Comment:
Good idea, but the functions SDK is not the only one we are working with

Reply:
Oh lol I didn't mean to push that, was more a personal note for when I got back to it😂 Would be good to have all the relevant links in the contributing.md if you do get a chance though!

Reply:
Done in https://github.com/IO-Design-Team/firebase_js_interop/pull/5